sw/device/silicon_creator/rom/e2e/ate/BUILD

```diff
@@ -0,0 +1,75 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@//rules:signing.bzl", "offline_presigning_artifacts", "offline_signature_attach")
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+load(
+    "//rules/opentitan:defs.bzl",
+    "fpga_params",
+    "opentitan_binary",
+    "opentitan_test",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+opentitan_binary(
+    name = "ate_gpio_toggle_test",
+    testonly = True,
```

Comment:
Shouldn't we had here something like:
dv = dv_params(
        rom = "@//sw/device/silicon_creator/rom:mask_rom",
    ),

Reply:
`dv_params` are only relevant for `opentitan_test` rules, not `opentitan_binary` rules.